File: JAVA API/crops/src/main/java/com/eagroservices/crops/pojos/Crop.java
package com.eagroservices.crops.pojos;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;


import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@Entity
@Table(name="crops")
public class Crop {

    @Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id;
	private String title;
    @Column(name = "imageurl")
	private String imageUrl;
	private Double rate;
    
}

File: API/Shipments/Entities/Crop.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Transflower.EAgroServices.Shipments.Entities;

[Table("crops")]
public class Crop
{
    [Key]
    [Column("id")]
    public int Id { get; set; }

    [Column("title")]
    public string? Title { get; set; }

    [Column("imageurl")]
    public string? ImageUrl { get; set; }

    [Column("rate")]
    public double Rate { get; set; }
}
